packages/salesforcedx-vscode-soql/src/editor/soqlEditorInstance.ts

```diff
@@ -22,18 +22,39 @@ const { OrgAuthInfo, channelService } = sfdxCoreExports;
 // This should be exported from soql-builder-ui
 export interface SoqlEditorEvent {
   type: string;
-  message?: string;
+  payload?: string | string[] | ToolingModelJson;
 }
 
 // This should be shared with soql-builder-ui
 export enum MessageType {
-  ACTIVATED = 'activated',
-  QUERY = 'query',
+  UI_ACTIVATED = 'ui_activated',
+  UI_SOQL_CHANGED = 'ui_soql_changed',
   SOBJECT_METADATA_REQUEST = 'sobject_metadata_request',
   SOBJECT_METADATA_RESPONSE = 'sobject_metadata_response',
   SOBJECTS_REQUEST = 'sobjects_request',
   SOBJECTS_RESPONSE = 'sobjects_response',
-  UPDATE = 'update'
+  TEXT_SOQL_CHANGED = 'text_soql_changed'
+}
+
+// TODO: move to shared module
+async function getConnection(): Promise<Connection> {
+  const usernameOrAlias = await OrgAuthInfo.getDefaultUsernameOrAlias(true);
```

Comment:
Passing true into this call will prompt the same message in the error below as a notification and in the output window. Do you still want to throw this error?